actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -577,7 +577,7 @@ public void setDeactivationTimeout(long deactivationTimeoutMs)
         logger.info("Starting Stage...");
         extensions = new ArrayList<>(extensions);
         startCalled = true;
-        if (state != null)
+        if (state != null && state != NodeCapabilities.NodeState.STOPPED)
```

Comment:
If `STOPPED` is the proper check then it would make sense to set the default value for `state` to `STOPPED`. In that case you can replace this with a check for `if(state != NodeState.STOPPED)`.